### DISCUSSION TITLE
Novo CodePen Projects

### DISCUSSION THREAD

**Discussion by kaueburiti** [frontendbr/forum#495] [Dicas]
> ### "Zero setup, full-featured front-end web development environment."

https://codepen.io/project/editor

O que acharam? 
Isso era exatamente o que faltava no CodePen para mim, achei genial a ideia e estou bem ansioso para testar de fato. Achei uma ótima ideia para "micro-projetos" que precisem de mais do que os 3 arquivos básicos de HTML/CSS/Javascript que o Codepen atualmente fornece.

Eu realmente adorei a proposta, e acredito que ajude o CodePen a ganhar mais relevância.
Não no sentido de um substituto de repositórios ou coisas do tipo, mas como um lugar para você exibir com muita facilidade esses "micro-projetos", como um que pretendo fazer de uma pequena interface por semana.

Nesse link você pode ler um pouco mais sobre todas as features: http://codepen.io/pro/projects/

**Comment by leandroruel** [comment]
é legal, ja usei serviços semelhantes que já tinham essa proposta a muito tempo.

**Comment by marabesi** [comment]
Eu sinto que essa feature era o que estava faltando mesmo, semana passada mesmo eu queria criar alguns arquivos separados só por questão de organização e não podia.

Agora ficou muito mais fácil e tranquilo :D

**Comment by kvnol** [comment]
Codepen :heart:

**Comment by akfzambrana** [comment]
Honestamente eu adorei essa opção de projetos, já to testando

**Comment by cassiocardoso** [comment]
Para compartilhar pequenos projetos é bem legal mesmo. O que mais curti dele até o momento é já vir com alguns projetos pré-configurados para usar como base para criar outros.

**Comment by felipefialho** [comment]
Acho que o Codepen conseguiu dar um passo adiante e deixou de ser só um local de experimentos para ser um lugar pra criar pequenos projetos com facilidade.

Bem legal.
